src/modules/auth/auth.service.ts

```diff
@@ -34,5 +43,126 @@ export class AuthService {
         return newUser
       }
 
-     
+      async login(loginDto: LoginDto) {
+        const user = await this.prismaService.users.findUnique({
+          where: { phoneNumber: loginDto.phoneNumber },
+          select: { 
```

Comment:
Do not need to get all those info, we only need what we use to create access_token